Error al ejecutar make deploy-dev

Cuando ejecuto make deploy-dev obtengo el siguiente error:

```
[4/4] Building fresh packages...
warning Error running install script for optional dependency: "/app/node_modules/optipng-bin: Command failed.
Exit code: 1
Command: node lib/install.js
Arguments: 
Directory: /app/node_modules/optipng-bin
Output:
⚠ spawn /app/node_modules/optipng-bin/vendor/optipng ENOENT
  ⚠ optipng pre-build test failed
  ℹ compiling from source
  ✖ Error: Command failed: /bin/sh -c ./configure --with-system-zlib --prefix=\"/app/node_modules/optipng-bin/vendor\" --bindir=\"/app/node_modules/optipng-bin/vendor\"

Checking for gcc...
Using pre-configured libpng...
Checking for system zlib...
./configure: error: missing zlib or incorrect zlib version
./configure: note: zlib version 1.2.1 or higher is required

    at /app/node_modules/bin-build/node_modules/execa/index.js:231:11
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Promise.all (index 0)"
```
Esto lo realicé por primera vez, es decir, es la primera vez que voy a levantar el sistema
en mi máquina, por lo que no debería tener ningún container levantado que pueda estar molestando. 

**Repoducción del error**
Pasos para reproducir el error:
1. Clonar el código
2. Ejecutar make deploy-dev
3. Ver el error en el paso 4/4

**Escritorio (complete la siguiente información):**
 - Debian Testing

Tambień observo que para la última version (2.2.0), si realizo make deploy-dev obtengo este error:

```
Creating docker_postgres_1 ... error
Creating docker_reactjs_1  ... 

Creating docker_reactjs_1  ... error

ERROR: for docker_reactjs_1  Cannot create container for service reactjs: invalid mount config for type "bind": bind source path does not exist: /home/eamanu/dev/eventol_release/eventoL-2.2.0/eventol/front/webpack-stats-local.json

ERROR: for postgres  Cannot create container for service postgres: invalid mount config for type "bind": bind source path does not exist: /home/eamanu/dev/eventol_release/eventoL-2.2.0/deploy/docker/db/postgres

ERROR: for reactjs  Cannot create container for service reactjs: invalid mount config for type "bind": bind source path does not exist: /home/eamanu/dev/eventol_release/eventoL-2.2.0/eventol/front/webpack-stats-local.json
ERROR: Encountered errors while bringing up the project.
make: *** [Makefile:187: deploy-dev] Error 1
```

el único container que se genera es  redis:

```
eamanu@debian:~/dev/eventol_release/eventoL-2.2.0$ docker ps
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS               NAMES
7c730b7046d9        redis:4.0.2         "docker-entrypoint.s…"   22 minutes ago      Up 15 minutes       6379/tcp            docker_redis_1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -5,7 +5,7 @@ FROM node:12.12.0-alpine as frontend
 
 ## Install system dependencies
 RUN apk --update add --no-cache \
-    git gcc make autoconf automake musl-dev \
+    git gcc make autoconf automake musl-dev zlib zlib-dev\
   && rm -rf /var/cache/apk/* /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 # Set working directory

